### PR TITLE
Completed javascript-local-storage exercise. -Paul Glorioso

### PR DESCRIPTION
**Pull request by PaulGlorioso**
ToDo

### CODE REVIEW

File: javascript-local-storage/data.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* exported todos */
+
+var todos = [];
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+todos = JSON.parse(previousTodosJSON);
```
Comment (BrettAlbright) [inline javascript-local-storage/data.js]: you want to make sure that the variable does not contain a null value before you assign it to the `todos` variable.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
looking good, one small update!

**Comment by PaulGlorioso** [comment]
ToDo2

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**